torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -7,6 +7,9 @@
 #include <torch/csrc/jit/tensorexpr/ir_printer.h>
 #include <torch/csrc/jit/tensorexpr/ir_simplifier.h>
 #include <torch/csrc/jit/tensorexpr/loopnest.h>
+#include "c10/core/ScalarType.h"
+#include "jit/tensorexpr/exceptions.h"
+#include "jit/tensorexpr/types.h"
```

Comment:
Oh, a minor bit of tidiness here, could you format the #includes as above, like using <...> and including the full torch/csrc/... path to the jit headers?  I kind of suspect those will make the internal buck build happy too.

Reply:
Indeed CI checking has reported the problem, and it has been fixed by a subsequent commit. (BTW, these header files seem to be inserted by my VSCode, I still don't know how they got there :<).